resources/dicts/patrols/general/medcat.json

```diff
@@ -479,147 +838,345 @@
-        "decline_text": "{PRONOUN/p_l/poss/CAP} companion stays with {PRONOUN/p_l/object} as silent support while {PRONOUN/p_l/subject} {VERB/p_l/work/works}.",
-        "chance_of_success": 30,
-        "success_outcomes": [
+            },
             {
-                "text": "With a respectful nod, p_l pauses, tucks {PRONOUN/p_l/poss} paws together neatly, and sits to converse with the StarClan cat.",
-                "exp": 30,
+                "text": "p_l shakes {PRONOUN/p_l/poss} head, telling app1 that {PRONOUN/app1/poss} dream is just a dream.",
+                "exp": 0,
                 "weight": 20,
-                "herbs": ["random_herbs"]
+                "relationships": [
+                    {
+                        "cats_to": ["p_l"],
+                        "cats_from": ["app1"],
+                        "mutual": false,
+                        "values": ["respect", "trust"],
+                        "amount": -5
+                    }
+                ]
             },
             {
-                "text": "The scent of a familiar pelt reaches p_l's nose - {PRONOUN/p_l/subject} {VERB/p_l/know/knows} this cat! With a friendly purr, {PRONOUN/p_l/subject} {VERB/p_l/turn/turns} to greet {PRONOUN/p_l/poss} friend from beyond the grave.",
-                "exp": 30,
-                "weight": 5,
-                "herbs": ["random_herbs"]
-            },
+                "text": "p_l eyes go wide, fur puffing up, and tells the apprentice that {PRONOUN/app1/poss}'s dream is a warning of death and destruction, thoroughly freaking app1 out.",
+                "exp": 0,
+                "weight": 20,
+                "art": "gen_train_argueMA",
+                "can_have_stat":["p_l"],
+                "stat_trait":[
+                    "gloomy",
+                    "sneaky",
+                    "strange",
+                    "insecure",
+                    "bold",
+                    "daring",
+                    "shameless",
+                    "oblivious"
+                ],
+                "relationships": [
+                    {
+                        "cats_to": ["p_l"],
+                        "cats_from": ["app1"],
+                        "mutual": false,
+                        "values": ["comfort", "respect", "trust"],
+                        "amount": -5
+                    }
+                ]
+            },
+            {
+                "text": "s_c stares at the apprentice with undisguised envy, desperately wishing {PRONOUN/s_c/subject} {VERB/s_c/were/was} the one to receive the vision. Glancing at the older cat nervously, app1 mumbles that it wasn't important.",
+                "exp": 0,
+                "weight": 20,
+                "stat_trait": [
+                    "ambitious", 
+                    "insecure", 
+                    "vengeful"
+                ],
+                "relationships": [
```

Comment:
i think we should gain some jealousy here as well

Reply:
Added! I also fixed the relationship block as it was supposed to be s_c not r_c.